USAGE.md

```diff
@@ -74,16 +61,44 @@ The GDL performs many checks on datasets. If any of the checks fail the dataset
 - Every metadata and asset URL must be readable by the GDL.
 - A dataset *may* refer to the same asset more than once. All references to the same asset must have the same multihash. That is, having a SHA-1 and a SHA-256 checksum for the same file will be considered invalid, even if both checksums are valid. This is to enable a simpler checksum validation.
 
-## Data Lake Endpoints Usage
+
+# Authentication and authorization
+Data Lake allows read/write access for users authorized by SAML identity provider
+(`DATALAKE_SAML_IDENTITY_PROVIDER_ARN`) configured during deployment time (see
+[README](README.md#aws-infrastructure-deployment-cdk-stack)).
+
+Example of AWS service account authentication and authorization in to Data Lake users role via
+Azure:
+
+- `~/.aws/config` file
+    ```
```

Comment:
List indentation in Markdown is three spaces.

Reply:
Indentation was fixed